text_dedup/minhash.py

```diff
@@ -99,94 +100,39 @@ def embed_func(
     ...     ],
     ...     dtype=np.uint64,
     ... ).T
-    >>> res = embed_func(content, idx, num_perm=num_perm, ngram_size=ngram_size, min_length=0, hashranges=hashranges, 
+    >>> res = embed_func(content, idx, num_perm=num_perm, ngram_size=ngram_size, min_length=0, hashranges=hashranges,
     ... permutations=PERMUTATIONS, hash_func=xxh3_hash)
```

Comment:
`xxh3_hash` becomes invalid since the import change.

Reply:
For local testing purposes, you can run `pytest --doctest-modules . --ignore deduplicate-text-datasets` for a quick sanity check.

Reply:
thanks for the tip! I have been using run testing (running the scripts to actually process the data itself)
but this would be helpful for quick tests!